### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by ZaiZu2**
Dependency to:
- https://github.com/ForloopAI/forloop_platform/pull/911

### CODE REVIEW

File: forloop_modules/globals/db_connection.py [hunk 1]
```diff
@@ -166,4 +166,41 @@ def decrypt_db_details(database: dict) -> DbDetails:
 
         return create_db_details_from_database_dict(db_dict=database)
     else:
-        raise ValueError("Private key not found in Redis.")
+        raise ValueError(f"Private key not found for Project:{database['project_uid']} in Redis.")
```
Comment (DovaX) [inline forloop_modules/globals/db_connection.py:169]: [Suggestion] database doesn't have to have "project_uid" key / None? - i.e. exception in exception

File: forloop_modules/globals/db_connection.py [hunk 2]
```diff
@@ -166,4 +166,41 @@ def decrypt_db_details(database: dict) -> DbDetails:
...
+    :rtype: dh.AbstractDb
+    """
+    types = [
+        db.python_database_type_mapping.get(python_type, "nvarchar(2047)")
+        for python_type in columns.values()
+    ]
+
+    # DBHydra expects ID to be the first column
+    column_names = list(columns.keys()) if 'id' in columns.keys() else ['id', *columns.keys()]
+    column_types = types if 'id' in columns.values() else ['int', *types]
```
Comment (DovaX) [inline forloop_modules/globals/db_connection.py:202]: please add these two as an issue in "dbhydra" so I can extend it in the future

Comment (DovaX) [reply forloop_modules/globals/db_connection.py:202]: its also quite hard to read as a ternary operator

**Review by DovaX — APPROVED** [review]

File: forloop_modules/globals/db_connection.py (hunk 1 above)
Comment (ZaiZu2) [reply forloop_modules/globals/db_connection.py:169]: This might be the case. I would keep this line nonetheless, as a simple assertion.

**Pull request merged**